pages/linux/pw-dot.md

```diff
@@ -0,0 +1,29 @@
+# pw-dot
+
+> Create `.dot` files of the PipeWire graph.
+> See also: `dot` for rendering graph.
+> More information: <https://docs.pipewire.org/page_man_pw-dot_1.html>.
+
+- Generate a graph to `pw.dot` file:
+
+`pw-dot`
+
+- Specify a filename showing all object types:
+
+`pw-dot --all --output {{path/to/file.dot}}`
```

Comment:
```suggestion
- Specify an output file, showing all object types:

`pw-dot --output {{path/to/file.dot}} --all`
```

I feel like this is clearer because it matches the order of the args. Also it's more accurately a path, not just a filename (although confusingly, it says "file name" in the man page).